### ISSUE TITLE
Idea for Assistant for first login in Admin panel

### ISSUE THREAD

**Issue by testt23** [luyadev/luya#1903]
Hi, I have idea for LUYA Assistant for first login in Admin panel

I can added in my admin panel , see example;
![3](https://user-images.githubusercontent.com/3694008/51790125-ed82dc80-2199-11e9-9fc8-ccca4280278b.png)
![4](https://user-images.githubusercontent.com/3694008/51790126-ed82dc80-2199-11e9-9d65-725527dee3a5.png)
![8](https://user-images.githubusercontent.com/3694008/51790127-ee1b7300-2199-11e9-8204-019617f71f38.png)
![1](https://user-images.githubusercontent.com/3694008/51790128-ee1b7300-2199-11e9-8772-1abca43c8777.png)

**Comment by nadar** [comment]
Yes, in general i like the idea, but in a first step i think we should only guide the user trough the admin ui elements: 

1. mainnav
2. collaps mainnav
3. search
4. where is profile
5. where is logout

stuff like this. Technically i think this would be a div which is display based on user_settings json value. So a person can start the "info assistant" from the profile "show assistant again" or when he does the first login attempt.

@TheMaaarc What about position of this wizzard?
@testt23 What have you used to generate those preview images?
